sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerContext.java

```diff
@@ -77,6 +78,7 @@
   private final String sql;
   private final DruidOperatorTable operatorTable;
   private final ExprMacroTable macroTable;
+  private final JoinableFactoryWrapper jfw;
```

Comment:
super super nit: tiny variable name doesn't seem very consistent with other names 😅

Reply:
My bad a new round is coming with all these changes (have them locally)